modules/tensor_mechanics/src/materials/ThermalFractureIntegral.C

```diff
@@ -0,0 +1,48 @@
+/****************************************************************/
```

Comment:
I like breaking this out from EshelbyTensor. Now that I look at this, though, it seems like we could just calculate this in the postprocessors.

Reply:
I tried to call getMaterialPropertyDerivative<RankTwoTensor> from InteractionIntegralPostprocessor and I am getting an error saying `no template named 'getMaterialPropertyDerivative'; did you mean 'getMaterialPropertyByName'?` even after including DerivativeInterfaceMaterial.h. Do I need to include something else to fix this error?

Reply:
You probably need to make the InteractionIntegralPostprocessor derive from DerivativeMaterialInterface.  Let's just leave this as you have it for the time being.